lidro/create_virtual_point/vectors/flatten_river.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+""" this function flattens the Z of the river
+"""
+import logging
+
+import geopandas as gpd
+import numpy as np
+
+from lidro.create_virtual_point.vectors.intersect_points_by_line import (
+    return_points_by_line,
+)
+
+
+def flatten_little_river(points: gpd.GeoDataFrame, line: gpd.GeoDataFrame):
+    """This function flattens the Z of the little rivers, because of
+        regression model doesn't work for these type of rivers.
+
+    Args:
+        points (gpd.GeoDataFrame): A GeoDataFrame containing points along Hydro's Skeleton
+        line (gpd.GeoDataFrame): A GeoDataFrame containing each line from Hydro's Skeleton
+
+    Returns:
+        float: Z of the river
+    """
+    # Inputs
+    gdf_points = return_points_by_line(points, line)
+    # Merge points and remove duplicates
+    all_points_knn = np.vstack(gdf_points["points_knn"].dropna().values)
+    if all_points_knn.size == 0:
+        logging.warning("For little river : error Z quartile")
```

Comment:
proposition d'erreur plus explicite : 
```suggestion
        logging.warning("For little river: no neighbor found to calculate Z quartile: set Z to 0")
```